### ISSUE TITLE
Add placeholder and aria-label for "open walkthrough"

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#29296]
Type: <b>Bug</b>

The "Open walk-through" dialog lacks a placeholder in the input box and an appropriate aria-label. 

VS Code version: Code 1.78.0 (252e5463d60e63238250799aef7375787f68b4ee, 2023-05-03T20:09:46.685Z)
OS version: Linux x64 6.3.11-1rodete1-amd64
Modes:
![placeholder lacking](https://github-com/microsoft/vscode/assets/2174646/389fc4b8-3b5f-4777-8249-0a02dd827ded)

Sandboxed: No
